fjord/mailinglist/admin.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib import admin
+
+from fjord.mailinglist.models import MailingList
+
+
+class MailingListAdmin(admin.ModelAdmin):
+    list_display = ('name', 'members')
+
+
+admin.site.register(MailingList, MailingListAdmin)
```

Comment:
There's also [the decorator](https://docs.djangoproject.com/en/1.8/ref/contrib/admin/#the-register-decorator) if you're into that.